### ISSUE TITLE
Add SSL for secure connections (https)

### ISSUE THREAD

**Issue by bhamodi** [optimizely/developers.optimizely.com#99]
Would be nice to have SSL on our dev site. :+1:  @tscanlin

**Comment by tscanlin** [comment]
Agreed and thanks for the suggestion @bhamodi but I think it only works on optimizely.github.io which I think we might want to keep open and not use for our developer docs. See: https://sheharyar.me/blog/free-ssl-for-github-pages-with-custom-domains/

I might be wrong though, I will look into it some more later this quarter if I have time but right now I don't see a pressing need. If you would like to help with this let me know.
